layouts/shortcodes/table.html

```diff
@@ -1,6 +1,11 @@
 {{ $hasCaption := isset .Params "caption" }}
 {{ $caption    := .Get "caption" }}
+{{ $sortable := .Get "sortable" }} 
 {{ $captionEl  := printf "<table><caption style=\"display: none;\">%s</caption>" $caption }}
+<!-- Check if 'sortable' is true, and if so, add the 'sortable-table' class -->    
+{{ if $sortable }}
+    {{ $captionEl = printf "<table class=\"sortable-table\"><caption style=\"display: none;\">%s</caption>" $caption }}
```

Comment:
This doesn't look right. This only makes tables sortable if they also have captions.

Reply:
Thanks for the review. Attempted to address this in the latest commit.